docs/current/tools.md

```diff
@@ -17,17 +17,78 @@
             </style>
 </head><article class="markdown-body">
 
-# Vaquero Validator
+# Vaquero Tools
 
-[Home](https://ciscocloud.github.io/vaquero-docs/) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master)
+[Home](https://ciscocloud.github.io/vaquero-docs/) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master) | 
+[Migrate](#vaquero-migrate) | [Preview](#vaquero-preview) | [Validator](#vaquero-validator)
 
+## Vaquero Migrate
```

Comment:
lets list some of the [use cases here](https://github.com/CiscoCloud/vaquero/issues/583)